game/tests/test_services.py

```
@@ -0,0 +1,243 @@
+        # Verify no-op when user is None
+        self.assertIsNone(update_opening_progress(None, "Sicilian Defense"))
+
+
+class TestActiveGameServices(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.user = User.objects.create_user(username="testuser", password="password")
+
+    def test_create_or_update_active_game(self):
+        request = self.factory.get("/")
+        middleware = SessionMiddleware(lambda req: None)
```

Ruff (0.15.18)

[warning] Unused lambda argument: `req`

(ARG005)